controllers.js

```diff
@@ -27,3 +35,23 @@ exports.getArticle = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getAllComments = (req, res, next) => {
+  const { article_id } = req.params;
+
+  if (isNaN(article_id)) {
+    return res.status(400).send({ msg: "Bad Request" });
+  }
```

Comment:
You don't need to manually check that the article_id is a number - if you let psql try and use it in the query you will get an error returned already. Catch it in the controller and see